examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -20,16 +20,33 @@ interface ITaskRowProps {
 const TaskRow: React.FC<ITaskRowProps> = (props: ITaskRowProps) => {
     const { task, deleteTask } = props;
     const priorityRef = useRef<HTMLInputElement>(null);
+    const [savedName, setSavedName] =  useState<string>(task.diffName);
+    const [savedPriority, setSavedPriority] = useState<number>(task.diffPriority);
+    const [savedDiffType, setSavedDiffType] = useState<string>(task.diffType);
     useEffect(() => {
         const updateFromRemotePriority = (): void => {
             if (priorityRef.current !== null) {
                 priorityRef.current.value = task.priority.toString();
             }
         };
+        const showSavedPriority = (): void => {
+            setSavedPriority(task.diffPriority);
+            setSavedDiffType(task.diffType);
+        }
+        const showSavedName = (): void => {
+            setSavedName(task.diffName);
+            setSavedDiffType(task.diffType);
+            console.log(savedName);
+            console.log(savedDiffType);
+        }
         task.on("priorityChanged", updateFromRemotePriority);
+        task.on("externalPriorityChanged", showSavedPriority);
+        task.on("externalNameChanged", showSavedName);
         updateFromRemotePriority();
         return (): void => {
             task.off("priorityChanged", updateFromRemotePriority);
+            task.on("externalPriorityChanged", showSavedPriority);
+            task.on("externalNameChanged", showSavedName);
         };
     }, [task]);
```

Comment:
I believe we need to update the effect deps here to reflect the new data you're using.

Reply:
Is this what you mean here? https://github.com/microsoft/FluidFramework/pull/13732/commits/0467fc9176871cb97399083c19a6a24f75c1a4b2

Reply:
Yup! Technically I think we also want to include the corresponding `set` functions as deps too (since we use them in the effect), but since they will strictly change when their corresponding values change, I think this is safe.